docs/source/commandline/commandline-all.rst

```diff
@@ -1526,63 +2118,92 @@ Debugging Options
 
 **-[no]verbose** / **-[no]v**
 
-	Displays internal diagnostic information. This information is very useful for debugging problems with your configuration. IMPORTANT: when reporting bugs, please run with **mame -verbose** and include the resulting information. The default is OFF (**-noverbose**).
+    Displays internal diagnostic information. This information is very useful
+    for debugging problems with your configuration.  IMPORTANT: when reporting
+    bugs, please run with **mame -verbose** and include the resulting
+    information.
+
+    The default is OFF (**-noverbose**).
 
 .. _mame-commandline-oslog:
 
 **-[no]oslog**
 
-	Output error.log data to the system debugger. The default is OFF (**-nooslog**).
+    Output error.log data to the system debugger.
+
+    The default is OFF (**-nooslog**).
 
 .. _mame-commandline-log:
 
 **-[no]log**
 
-	Creates a file called error.log which contains all of the internal log messages generated by the MAME core and system drivers. This can be used at the same time as **-log** to output the log data to both targets as well. The default is OFF (**-nolog**).
+    Creates a file called error.log which contains all of the internal log
+    messages generated by the MAME core and system drivers.  This can be used at
+    the same time as **-log** to output the log data to both targets as well.
+
+    The default is OFF (**-nolog**).
 
 .. _mame-commandline-debug:
 
 **-[no]debug**
 
-	Activates the integrated debugger. By default, the debugger is entered by pressing the tilde (**~**) key during emulation. It is also entered immediately at startup. The default is OFF (**-nodebug**).
+    Activates the integrated debugger.  By default, the debugger is entered by
+    pressing the tilde (**~**) key during emulation. It is also entered
+    immediately at startup.
+
+    The default is OFF (**-nodebug**).
 
 .. _mame-commandline-debugscript:
 
 **-debugscript** *<filename>*
 
-	Specifies a file that contains a list of debugger commands to execute immediately upon startup. The default is ``NULL`` (*no commands*).
+    Specifies a file that contains a list of debugger commands to execute
+    immediately upon startup.
+
+    The default is ``NULL`` (*no commands*).
 
 .. _mame-commandline-updateinpause:
 
 **-[no]update_in_pause**
 
-	Enables updating of the main screen bitmap while the system is paused. This means that the VIDEO_UPDATE callback will be called repeatedly during pause, which can be useful for debugging. The default is OFF (**-noupdate_in_pause**).
+    Enables updating of the main screen bitmap while the system is paused.  This
+    means that the VIDEO_UPDATE callback will be called repeatedly during pause,
```

Comment:
The macros are going away - replace "VIDEO_UPDATE" with "video update" - it's plain English anyway.  Also, "during pause" is ugly, how about "while emulation is paused"?

Reply:
Done.